Project.toml

```diff
@@ -10,9 +10,6 @@ MathOptInterface = "b8f27783-ece8-5eb3-8dc8-9495eed66fee"
 Random = "9a3f8284-a2c9-5f02-9a11-845980a1fd5c"
 
 [compat]
-Cbc = "≥ 0.6.0"
-JuMP = "~0.22"
-MathOptInterface = "~0.10"
 julia = "1"
```

Comment:
You should keep a compat for JuMP and MOI. Otherwise, releasing won't be possible

Reply:
thanks, realized that the hard way with Hypatia last night.